source/layout.erb

```diff
@@ -17,21 +17,18 @@
 
     <%= yield_content :outside_wrapper %>
 
-    <div class="container">
+    <main class="container">
       <%= partial "sidebar" %>
 
-      <div id="content" class="has-sidebar">
-        <div class="chapter">
+      <div class="content">
+        <article>
           <%= chapter_heading %>
           <hr>
-
           <%= yield %>
-
-
           <%= chapter_links %>
-        </div>
+        </article>
       </div>
```

Comment:
What are your thoughts on updating 23-30 to something like:

```erb
<article class="content">
  <%= chapter_heading %>
    <hr>
    <%= yield %>
  <%= chapter_links %>
</article>
```

(combined the `div` and `article` tag together)